MokManager.c

```diff
@@ -536,10 +536,11 @@ static void show_efi_hash (EFI_GUID Type, void *Mok, UINTN MokSize)
 	menu_strings[i+1] = NULL;
 
 	while (key_num < hash_num) {
-		key_num = console_select((CHAR16 *[]){ L"[Hash List]", NULL },
+		int rc;
+		key_num = rc = console_select((CHAR16 *[]){ L"[Hash List]", NULL },
```

Comment:
Weird, Github shows in the list of commits for this PR #100 as if this commit comes after 0c89c6d9a228 ("Get rid of all the places we cast to (CHAR16 *[])"), but I see in the [commits of your coverity branch]( https://github.com/vathpela/mallory/commits/coverity ) that's the opposite.

And now I see that the order is wrong for other commits too but didn't notice because were unrelated.